crates/eframe/src/epi/mod.rs

```diff
@@ -455,6 +455,9 @@ pub struct NativeOptions {
     /// }
     /// ```
     pub app_id: Option<String>,
+
+    /// Enable/disable IME at startup.
+    pub allow_ime: bool,
```

Comment:
Please describe this a bit more.

If `true`, will an on-screen-keyboard usually be shown for touch-screens on start? Will it hide once the user stops editing text?

If `false`, will it still show up when you start editing text? Will it hide once you stop editing the text?

Reply:
> If `true`, will an on-screen-keyboard usually be shown for touch-screens on start? Will it hide once the user stops editing text?

You have to manually hide the keyboard on Phosh.

> If `false`, will it still show up when you start editing text? Will it hide once you stop editing the text?

It will not automatically show the keyboard when you enter a text edit. You have to manually show it.

Adding this to my code will automatically show/hide the keyboard on Phosh:
```Rust
          let response = ui.text_edit_singleline(&mut self.text);
          if response.gained_focus() {
            frame.allow_ime(true);
          } else if response.lost_focus() {
            frame.allow_ime(false);
          }
```

I added `allow_ime` to `NativeOptions` in case someone wants to keep the previous behavior.

Reply:
I may be that enabling and disabling IME shows and hides the keyboard for other OS' but I really don't have a way to test that.

Reply:
It seems to me the desirable behavior is to have one flag that controls wether or not the on-screen-keyboard shows on startup (with `false` being a reasonable default).

And then, regardless of that flag, always show the on-screen-keyboard when the user clicks a text field

Reply:
That's my ultimate goal but egui is rather disconnected from the windowing system, so this was the best that I could come up with.